### PR TITLE
Button Displayed When room is selected

### PR DESCRIPTION
**Pull request by MlulekiN**

### CODE REVIEW

File: Feature/SelectedRoomBookNow.feature [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Feature: Selected Room Book Now Button
```
Comment (benedictnkosi) [inline Feature/SelectedRoomBookNow.feature:1]: learn how to name feature

File: Feature/SelectedRoomBookNow.feature [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+  Scenario Outline: Clicking the displayed Book Now Button
```
Comment (benedictnkosi) [inline Feature/SelectedRoomBookNow.feature]: learn how to name scenario

**Review by benedictnkosi — APPROVED** [review]

File: Feature/SelectedRoomBookNow.feature (hunk 2 above)
Comment (MlulekiN) [reply Feature/SelectedRoomBookNow.feature]: Done

File: Feature/SelectedRoomBookNow.feature (hunk 1 above)
Comment (MlulekiN) [reply Feature/SelectedRoomBookNow.feature:1]: Done

**Pull request merged**